cdk/README.md

```diff
@@ -1,6 +1,13 @@
-# Welcome to your CDK TypeScript project!
+# Getting Started
+1. Navigate to the cdk directory in terminal
+2. `npm install monocdk`
```

Comment:
Actually now that I think about it, what is the point of doing this? If monocdk is provided in the `package.json`, then shouldn't someone using this only need to `npm install`? If it's for the CLI, don't they need to install it globally? 🤔